save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ProjectController.kt

```diff
@@ -72,19 +74,53 @@ class ProjectController(private val projectService: ProjectService,
      * FixMe: requires 'write' permission, because now we rely on this endpoint to load `ProjectView`
      *
      * @param name name of project
-     * @param owner owner of project
      * @param authentication
-     * @return project by name and owner
+     * @param organizationId
+     * @return project by name and organization
      * @throws ResponseStatusException
      */
     @GetMapping("/get")
     @PreAuthorize("hasRole('VIEWER')")
     @Suppress("UnsafeCallOnNullableType")
-    fun getProjectByNameAndOwner(@RequestParam name: String,
-                                 @RequestParam owner: String,
-                                 authentication: Authentication,
+    fun getProjectByNameAndOrganization(@RequestParam name: String,
+                                        @RequestParam organizationId: Long,
+                                        authentication: Authentication,
     ): Mono<Project> = Mono.fromCallable {
-        projectService.findByNameAndOwner(name, owner)
+        val organization = organizationService.getOrganizationById(organizationId)
+        projectService.findByNameAndOrganization(name, organization)
+    }
+        .map {
```

Comment:
This method is just a copy paste of logic from `getProjectByNameAndOrganizationName`, the single difference, that in the begging, in one case we take `organization` by name, in other by `id` let's extract common logic into one method

Reply:
Fixed